### ISSUE TITLE
Tests are failing - potentially from napari 0.5?

### ISSUE THREAD

**Issue by alisterburt** [napari-threedee/napari-threedee#250]
```python
=================================== FAILURES ===================================
  ____________________ test_add_point_on_plane_same_scale_3d _____________________
  
  viewer_with_plane_and_points_3d = Viewer(camera=Camera(center=(13.5, 13.5, 13.5), zoom=20.357142857142854, angles=(0.0, 0.0, 90.0), perspective=0.0, mou...nction dims_scroll at 0x7fd45fe31670>], _persisted_mouse_event={}, _mouse_drag_gen={}, _mouse_wheel_gen={}, _keymap={})
  
      def test_add_point_on_plane_same_scale_3d(viewer_with_plane_and_points_3d):
          """Test adding points on a plane when the layers have same non-[1, 1, 1] scale"""
          viewer = viewer_with_plane_and_points_3d
          scale = (2, .5, .5)
          points_layer = viewer.layers['Points']
          points_layer.scale = scale
          plane_layer = viewer.layers['blobs_3d']
          plane_layer.scale = scale
          viewer.dims.ndisplay = 3
      
          # the event is in world (scaled) coordinates
          event = MockMouseEvent(
              modifiers = ['Alt']
          )
          viewer.cursor.position = (14, 14, 14)
          viewer.camera.set_view_direction((1, 0, 0))
      
          # plane position is (14, 14, 14), in data coordinates
          on_mouse_alt_click_add_point_on_plane(
              viewer=viewer_with_plane_and_points_3d,
              event=event,
              points_layer=points_layer,
              image_layer=plane_layer,
          )
  >       assert len(points_layer.data) == 1
  E       AssertionError: assert 0 == 1
  E        +  where 0 = len(array([], shape=(0, 3), dtype=float64))
  E        +    where array([], shape=(0, 3), dtype=float64) = <Points layer 'Points' at 0x7fd42863a910>.data
  
  src/napari_threedee/utils/_tests/test_mouse_callbacks.py:93: AssertionError
  ____________________ test_add_point_on_plane_same_scale_4d _____________________
  
  viewer_with_plane_and_points_4d = Viewer(camera=Camera(center=(13.5, 13.5, 13.5), zoom=20.357142857142854, angles=(0.0, 0.0, 90.0), perspective=0.0, mou...nction dims_scroll at 0x7fd45fe31670>], _persisted_mouse_event={}, _mouse_drag_gen={}, _mouse_wheel_gen={}, _keymap={})
  
      def test_add_point_on_plane_same_scale_4d(viewer_with_plane_and_points_4d):
          """Test adding points on a plane when the layers have same non-[1, 1, 1] scale"""
          viewer = viewer_with_plane_and_points_4d
          scale = (1, 2, .5, .5)
          points_layer = viewer.layers['Points']
          points_layer.scale = scale
          plane_layer = viewer.layers['blobs_4d']
          plane_layer.scale = scale
          viewer.dims.ndisplay = 3
      
          # set the dims point
          slice_index = 12
          viewer.dims.set_current_step(0, slice_index)
      
          # the event is in world (scaled) coordinates
          event = MockMouseEvent(
              modifiers = ['Alt']
          )
          viewer.cursor.position = (0, 14, 14, 14)
          viewer.camera.set_view_direction((1, 0, 0))
      
          # plane position is (14, 14, 14), in data coordinates
          on_mouse_alt_click_add_point_on_plane(
              viewer=viewer_with_plane_and_points_4d,
              event=event,
              points_layer=points_layer,
              image_layer=plane_layer,
          )
  >       assert len(points_layer.data) == 1
  E       AssertionError: assert 0 == 1
  E        +  where 0 = len(array([], shape=(0, 4), dtype=float64))
  E        +    where array([], shape=(0, 4), dtype=float64) = <Points layer 'Points' at 0x7fd4[283](https://github.com/napari-threedee/napari-threedee/actions/runs/9909362681/job/27377257968?pr=248#step:7:287)40be0>.data
  
  src/napari_threedee/utils/_tests/test_mouse_callbacks.py:131: AssertionError
  ____________________ test_light_dir_unchanged_when_disabled ____________________
  
  make_napari_viewer = <function make_napari_viewer.<locals>.actual_factory at 0x7fd4[284](https://github.com/napari-threedee/napari-threedee/actions/runs/9909362681/job/27377257968?pr=248#step:7:288)ee700>
  lone_triangle = (array([[-1,  0,  0],
         [ 1,  0,  0],
         [ 0,  1,  0]]), array([[0, 1, 2]], dtype=int16))
  
      def test_light_dir_unchanged_when_disabled(make_napari_viewer, lone_triangle):
          viewer = make_napari_viewer()
          viewer.dims.ndisplay = 3
          layer = viewer.add_surface(lone_triangle)
          visual = get_napari_visual(viewer=viewer, layer=layer)
      
          lightting_control = LightingControl(viewer)
          lightting_control.set_layers([layer])
      
          inital_camera_angles = (0, 0, 90)
          viewer.camera.angles = inital_camera_angles
          inital_light_dir = visual.node.shading_filter.light_dir
          viewer.camera.angles = (0, 0, -90)
      
  >       assert visual.node.shading_filter.light_dir == inital_light_dir
  E       assert (-1.0, -0.999...0000000000002) == (-1.0, 1.0, 1.0)
  E         
  E         At index 1 diff: -0.9999999999999999 != 1.0
  E         
  E         Full diff:
  E           (
  E               -1.0,
  E         -     1.0,
  E         -     1.0,
  E         +     -0.9999999999999999,
  E         +     -1.0000000000000002,
  E           )
  
  src/napari_threedee/visualization/_tests/test_lighting_control.py:47: AssertionError
```

**Comment by alisterburt** [comment]
closed by #251

**Comment by psobolewskiPhD** [comment]
Looks like this was a duplicate of #242 I will close that one too, linking to the PR
